packages/components/src/styles/components/app-frame.scss

```diff
@@ -16,32 +16,47 @@
   grid-template-rows: auto 1fr auto;
   grid-template-columns: auto 1fr;
   min-height: 100vh;
+}
+
+// OPTIONS
 
-  // prevent interaction with AppHeader when SideNav is open/expanded
-  &:has(.hds-side-nav--is-not-minimized) .hds-app-header--is-mobile {
-    pointer-events: none;
+// Make header position sticky/fixed if is-header-fixed is set and the viewport height is greater than 480px
+@media (height >= 480px) {
+  .hds-app-frame--is-header-fixed {
+    .hds-app-frame__header {
+      position: sticky;
+      top: 0;
+      right: 0;
+      left: 0;
+    }
+  
+    // Modify sidenav layout when used together with fixed app-header
+    .hds-app-frame__sidebar {
+      top: var(--token-app-header-height);
+      height: calc(100vh - var(--token-app-header-height));
+      min-height: calc(100vh - var(--token-app-header-height));
+    }
   }
 }
 
-
 // FRAME'S CONTAINERS
 
 .hds-app-frame__header {
-  z-index: 7;
+  z-index: 21; // needs to stay above the main content & sidebar
   grid-area: header;
-
-  // NOTE: May not need to qualify this with .hds-app-header as it may not matter if AppHeader is not present
-  &:has(.hds-app-header) {
-    position: sticky;
-    top: 0;
-    right: 0;
-    left: 0;
-  }
 }
 
 .hds-app-frame__sidebar {
-  z-index: 6;
+  position: sticky;
```

Comment:
See comment above. For me this should be an `opt-in` behaviour.

Reply:
Leaving this alone for now until there's more discussion and a firmer decision.